json_schema/system/links.json

```
@@ -80,7 +80,7 @@
         }
       }
     },
-    "link": {
+    "experiment_link": {
```

I would use

```suggestion
    "process_link": {
```

"experiment" may be overly restrictive. After all, each link represents a process.

---

process link seems ok to me